skorch/net.py

```diff
@@ -1599,6 +1609,100 @@ def __setstate__(self, state):
 
         self.__dict__.update(state)
 
+    def _register_attribute(
+            self,
+            name,
+            prefixes=True,
```

Comment:
Do we call `_register_attribute` with `prefixes=False`?

We also always call this with `cuda_dependent_attributes=True`. I guess there are no torch components that are not also `cuda_dependent_attributes`?

Reply:
Yes, it could be that this is one of these cases of YAGNI. My idea was to provide a more general "setter" for prefixes and cuda dependent attributes that could be used in other places as well, where it might not make sense to change both at the same time.